mw-webapp/src/logic/reportsTable/DayReportsTable.tsx

```diff
@@ -0,0 +1,54 @@
+import {useEffect, useState} from "react";
+import {useParams} from "react-router-dom";
+import {Button} from "src/component/button/Button";
+import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
+import {QueryParamTypes} from "src/logic/QueryParamTypes";
+import {columns} from "src/logic/reportsTable/columns";
+import {ReportsTable} from "src/logic/reportsTable/ReportsTable";
+import {DayReport} from "src/model/businessModel/DayReport";
+
+/**
+ * Render table with dayReports of specific way
+ */
+export const DayReportsTable = () => {
+  const {uuid} = useParams<QueryParamTypes["uuid"]>();
```

Comment:
this is a specific parameter for specific page, so it should be move to the place where we describe all pages data, agreed?) Ideally typesafe way

Reply:
DOn't understant what need to move out. Type of queryParamTypes?

Reply:
currently "uuid" is just string, but it should be connected with page params

Reply:
currently "uuid" is just string, but it should be connected with page params